content/en/continuous_integration/setup_pipelines/gitlab.md

```diff
@@ -26,24 +35,32 @@ sudo gitlab-rails runner "Feature.enable(:datadog_ci_integration)"
 For installations from source:
 
 {{< code-block lang="shell" >}}
-sudo -u git -H bundle exec rails runner -e production "Feature.enable(:datadog_ci_integration)"
+sudo -u git -H bundle exec rails runner \
+  -e production \
+  "Feature.enable(:datadog_ci_integration)"
 {{< /code-block >}}
 
 For [Kubernetes deployments][5]:
 
 {{< code-block lang="shell" >}}
-kubectl exec -it <task-runner-pod-name> -- /srv/gitlab/bin/rails runner "Feature.enable(:datadog_ci_integration)"
+kubectl exec -it <task-runner-pod-name> -- \
+  /srv/gitlab/bin/rails runner "Feature.enable(:datadog_ci_integration)"
 {{< /code-block >}}
 
-Once the feature flag is enabled, configure the integration on a per project basis by going to **Settings > Integrations > Datadog** for each project you want to instrument. Mark it as active and paste a valid [API key][6].
+## Configuring the Datadog integration
+
+Configure the integration on a per project or group basis by going to **Settings > Integrations > Datadog** for each project or group you want to instrument. You can also activate the integration at the GitLab [instance level][10] by going to **Admin > Settings > Integrations > Datadog**, which will be applied to all existing and future projects.
 
-__Note:__ Only `datadoghq.com` is currently supported.
+Fill in the integration configuration settings:
 
-There are a couple of optional parameters:
-- _service_: Sets the service name for all traces for this GitLab instance. Defaults to `gitlab-ci`. Useful if you're using more than one GitLab instance.
-- _env_: Sets the environment tag for all traces for this GitLab instance. Useful in advanced setups where grouping GitLab instances per environment is required.
+* **Active** enables the integration.
+* **Datadog site** specifies which [Datadog site][11] to send data to. Please note that only `datadoghq.com` is currently supported.
+* **API URL** (optional) allows overriding the API URL used for sending data directly, only used in advanced scenarios.
+* **API key** specifies which API key to use when sending data. You can generate one in the [APIs tab][6] of the Integrations section on Datadog.
+* **Service** (optional) specifies which service name to attach to each span generated by the integration. Use this to differentiate between GitLab instances.
+* **Env** (optional) specifies which environment (`env` tag) to attach to each span generated by the integration. Use this to differentiate between groups of GitLab instances (i.e. staging vs production).
 
-Test the integration with the *Test settings* button. After it's successful, click *Save changes* to finish the integration set up.
+You can test the integration with the **Test settings** button. After it’s successful, click **Save changes** to finish the integration set up.
```

Comment:
The Test button is only available at project-level config. Maybe we should reflect that

Reply:
```suggestion
You can test the integration with the **Test settings** button. After it's successful, click **Save changes** to finish the integration set up.
```